tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -68,6 +73,145 @@ def test_efs_same_az(region, scheduler, pcluster_config_reader, clusters_factory
     _test_efs_correctly_shared(remote_command_executor, mount_dir, scheduler_commands)
 
 
+@pytest.mark.dimensions("ap-northeast-2", "c5.xlarge", "alinux", "awsbatch")
```

Comment:
let's use alinux2 for batch and centos8 for slurm, because we're going to deprecate alinux